i3-policy-store.yaml

```diff
@@ -793,18 +793,35 @@ components:
         - $ref: '#/components/schemas/Condition'
         - type: object
           required:
-            - tag
+            - elements
```

Comment:
I propose an array of objects (an object contains: tag, operator, content) to be consistent with LocationCondition

Reply:
Done